internal/services/eventgrid/eventgrid_domain_resource.go

```diff
@@ -140,6 +150,17 @@ func resourceEventGridDomain() *pluginsdk.Resource {
 				},
 			},
 
+			"minimum_tls_version": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  string(domains.TlsVersionOnePointTwo),
+				ValidateFunc: validation.StringInSlice([]string{
+					string(domains.TlsVersionOnePointZero),
+					string(domains.TlsVersionOnePointOne),
+					string(domains.TlsVersionOnePointTwo),
```

Comment:
These values will be retired soon so we should not expose them.
```suggestion
				ValidateFunc: validation.StringInSlice([]string{
					string(domains.TlsVersionOnePointTwo),
```